config/site_settings.yml

```diff
@@ -1614,6 +1614,18 @@ search:
       ro: true
       sk: true
       tr_TR: true
+  category_search_priority_very_low_weight:
```

Comment:
I am uneasy with all these toggles in the UX it feels like something that will just cause confusion to admins, can we start off with just having these as hidden site settings?